manifests/broker.yml

```diff
@@ -12,6 +12,7 @@ meta:
     api_url:    (( concat "https://api." meta.cf.system_domain ))
     username:   (( vault meta.cf.exodus_path ":admin_username" ))
     password:   (( vault meta.cf.exodus_path ":admin_password" ))
+    skip_ssl_validation: true
```

Comment:
this should be exposed as a parameter and defaulted to false. for CF APIs that serve signed certs, verifying their contents is good :)